Cannot redeclare block-scoped variable '__DEBUG_BUILD__'.

When building my project, I get the following error:

```
yarn run v1.22.19
$ tsc
node_modules/@sentry/types/types/globals.d.ts:2:11 - error TS2451: Cannot redeclare block-scoped variable '__DEBUG_BUILD__'.

2     const __DEBUG_BUILD__: boolean;
            ~~~~~~~~~~~~~~~

  node_modules/winston-transport-sentry-node/node_modules/@sentry/types/types/globals.d.ts:2:11
    2     const __DEBUG_BUILD__: boolean;
                ~~~~~~~~~~~~~~~
    '__DEBUG_BUILD__' was also declared here.

node_modules/winston-transport-sentry-node/node_modules/@sentry/types/types/globals.d.ts:2:11 - error TS2451: Cannot redeclare block-scoped variable '__DEBUG_BUILD__'.

2     const __DEBUG_BUILD__: boolean;
            ~~~~~~~~~~~~~~~

  node_modules/@sentry/types/types/globals.d.ts:2:11
    2     const __DEBUG_BUILD__: boolean;
                ~~~~~~~~~~~~~~~
    '__DEBUG_BUILD__' was also declared here.


Found 2 errors.

error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

In my package.json, I have 2 Sentry-related `dependencies`:

```
...
    "@sentry/node": "7.14.2",
    "winston-transport-sentry-node": "2.6.0",
...
```

I looked at `winston-transport-sentry-node`'s dependencies, and it seems like it has a direct dependency on `@sentry/node`: https://github.com/aandrewww/winston-transport-sentry-node/blob/master/package.json#L38

**Suggestion:** `@sentry/node` should be included as a `peerDependencies` instead of a regular `dependencies`

**Temporary fix:** I had to remove my dependency on `@sentry/node` from my project. However, now I'm stuck with the `@sentry/node` version that is provided by `winston-transport-sentry-node`

--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "winston-transport-sentry-node",
   "description": "@sentry/node transport for the winston v3 logger",
-  "version": "2.7.0",
+  "version": "2.7.1",
   "author": "Andrew Avdeev <andrewww.avdeev@gmail.com>",
   "keywords": [
     "logger",
@@ -35,7 +35,6 @@
     "node": ">= 8.10.0"
   },
   "dependencies": {
-    "@sentry/node": "^7.1.1",
     "triple-beam": "^1.3.0",
     "tslib": "^2.3.1",
     "winston": "^3.3.3",
@@ -52,5 +51,8 @@
     "eslint": "^7.32.0",
     "mocha": "^9.1.2",
     "typescript": "^4.4.3"
+  },
+  "peerDependencies": {
+    "@sentry/node": "^7.1.1"
   }
 }

